app/content/drill/[id]/leaderboard.js

```diff
@@ -61,6 +62,12 @@ export default function Leaderboard() {
     error: drillError,
   } = useDrillInfo({ drillId });
 
+  const {
+    data: allTimeInfo,
+    isLoading: allTimeRecordIsLoading,
+    error: allTimeRecordError,
+  } = useAllTimeRecords(drillId);
```

Comment:
can you make this useAllTimeRecords({drillId})? I've changed all hooks to always take objects so it will make it consistent with other hooks and with the queryKey.